packages/workbox-strategies/NetworkFirst.mjs

```diff
@@ -181,6 +186,7 @@ class NetworkFirst {
     try {
       response = await fetchWrapper.fetch(
         event.request,
+        this._fetchOptions,
```

Comment:
There's a bug in the current codebase—the plugins array was being passed as the second parameter in this invocation, instead of the third parameter.

(The joy of positional arguments...)